perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/widgets/items/validators/CheckboxValidator.java

```diff
@@ -24,7 +24,7 @@ public boolean validateLocal(Checkbox checkbox) {
 			} else {
 				// single checkbox - prefer own error message
 				setResult(Result.EMPTY);
-				checkbox.setRawStatus(getErrorMsgOrDefault(checkbox), ValidationState.ERROR);
+				checkbox.setRawStatus(getTransl().cantBeEmptySingleCheckBox(), ValidationState.ERROR);
```

Comment:
Now you prevent the possibility to display custom error message (eg. "You must agree to the AUP"). You should probably check if returned error message is the same as the default message and in that case replace it with your message. I'm not sure if there is getErrorMsg() without the default, then you could just check if it returns empty/null message and use your message instead.

Reply:
Should now be fixed as suggested.